lib/improver/cli/spot_extract.py

```diff
@@ -213,9 +212,14 @@ def main(argv=None):
     # Check whether a lapse rate cube has been provided and we are dealing with
     # temperature data.
     if (args.temperature_lapse_rate_filepath and
-            diagnostic_cube.name() == "air_temperature"):
+            result.name() == "air_temperature"):
```

Comment:
Was this a bug before?

Reply:
Yes. As the CLI is only setup to handle lapse rate adjustment of "deterministic" quantities, it cannot adjust a cube of 'probability_of_air_temperature_above...', hence the full name was used as the test. The last PR to modify this CLI added the percentile extraction step, which means a probability cube can now be converted to a single percentile (e.g. 50th percentile) cube before reaching this test. We can adjust such a cube (though we should be careful about what this means), so the test has been modified to check the name that results from the percentile extraction, rather than the original input name.